extensions/modules/src/org/exist/xquery/modules/xslfo/ApacheFopProcessorAdapter.java

```diff
@@ -126,7 +126,9 @@ private FOUserAgent setupFOUserAgent(final FOUserAgent foUserAgent, final Proper
      */
     private ResourceResolver getResourceResolver(final DBBroker broker, final String baseUri) {
         final ResourceResolverFactory.SchemeAwareResourceResolverBuilder builder = ResourceResolverFactory.createSchemeAwareResourceResolverBuilder(ResourceResolverFactory.createDefaultResourceResolver());
-        final URIResolverAdapter uriResolver = new URIResolverAdapter(new ExistSchemeRewriter(new EXistURIResolver(broker, baseUri)));
+        final URIResolverAdapter uriResolver = new URIResolverAdapter(
+            new ExistSchemeRewriter(new EXistURIResolver(broker.getBrokerPool(), baseUri))
+        );
         builder.registerResourceResolverForScheme("exist", uriResolver);
```

Comment:
@dizzzz better to ask @adamretter why it here at all. He did point [here](https://github.com/eXist-db/exist/pull/1340) that we did/should not use `exist:///`, so this must cleaned up.

If you want it to be fixed open issue for it (to remember).